assets/src/blocks/Timeline/TimelineEditor.js

```diff
@@ -0,0 +1,196 @@
+	{label: 'Top', value: 'top'},
+]
+
+let url_desc  = __(
+	'Enter the URL of the Google Sheets spreadsheet containing your timeline data.',
+	'p4ge'
+);
+url_desc += '<br><a href="https://timeline.knightlab.com/#make" target="_blank" rel="noopener noreferrer">';
+url_desc += __(
+	'See the TimelineJS website for a template GSheet.',
+	'p4ge'
+);
+url_desc += '</a><br>';
+url_desc += __(
+	'Copy this, add your own timeline data, and publish to the web.',
+	'p4ge'
+);
+url_desc += '<br>';
+url_desc += __(
+	"Once you have done so, use the URL from your address bar (not the one provided in Google's 'publish to web' dialog).",
+	'p4ge'
+);
+
+const renderEdit = (attributes, toAttribute) => {
+	return (
+    <Fragment>
+      <InspectorControls>
+        <PanelBody title={__('Setting', 'planet4-blocks-backend')}>
+					<URLInput
+						label={__('Google Sheets URL', 'p4ge')}
+						placeholder={__('Enter URL', 'p4ge')}
+						help={ <RawHTML>{url_desc}</RawHTML> }
+						value={attributes.google_sheets_url}
+						onChange={toAttribute('google_sheets_url')}
+					/>
+
+					<SelectControl
+						label={__('Language', 'p4ge')}
+						value={attributes.language}
+						options={languages}
+						onChange={toAttribute('language')}
+					/>
+
+					<SelectControl
+						label={__('Timeline navigation position', 'p4ge')}
+						value={attributes.timenav_position}
+						options={position}
+						onChange={toAttribute('timenav_position')}
+					/>
+
+					<CheckboxControl
+						label={__('Start at end', 'p4ge')}
+						help={__('Begin at the end of the timeline', 'p4ge')}
+						value={attributes.start_at_end}
+						checked={attributes.start_at_end}
+						onChange={toAttribute('start_at_end')}
+					/>
+
+				</PanelBody>
+			</InspectorControls>
+		</Fragment>
+	)
+}
+
+const renderView = ({ attributes }, toAttribute) => {
+  return (
+    <Fragment>
+      <Tooltip text={__('Edit text', 'planet4-blocks-backend')}>
+        <header className="articles-title-container">
+          <RichText
+            tagName="h2"
+            className="page-section-header"
+            placeholder={__('Enter title', 'planet4-blocks-backend')}
+            value={attributes.timeline_title}
+            onChange={toAttribute('timeline_title')}
+            keepPlaceholderOnFocus={true}
+            withoutInteractiveFormatting
+            characterLimit={40}
+            multiline="false"
+          />
+        </header>
+      </Tooltip>
+      <RichText
+        tagName="p"
+        className="page-section-description"
+        placeholder={__('Enter description', 'planet4-blocks-backend')}
+        value={attributes.description}
+        onChange={toAttribute('description')}
+        keepPlaceholderOnFocus={true}
+        withoutInteractiveFormatting
+        characterLimit={200}
+      />
+      <TimelineFrontend {...attributes} />
+		</Fragment>
+	)
+}
+
+export const TimelineEditor = (props) => {
+  const { isSelected, attributes, setAttributes } = props;
```

Comment:
You could destructure these directly in the declaration to avoid an extra line 🙂 aka:
```
export const TimelineEditor = ({ isSelected, attributes, setAttributes }) => {
```